### DISCUSSION TITLE
DexCollectionService: Error receiving too much data

### DISCUSSION THREAD

**Discussion by JohhnyP** [NightscoutFoundation/xDrip#2342] [Q&A] [answered]
Getting the following error, on master device often, resulting in missed readings.

![20220917_162622](https://user-images.githubusercontent.com/11662759/190859405-f1fdc935-c70c-4717-9de1-c9c9a41bf20e.jpg)

Is there something I can do to fix it? Is it a known bug?

Device: Samsung A40
Android Version: 11 (Jul 1, 2022)
XDrip Version: e557cba-2022.05.19 (2122051915)
Bridge: Libre1 with MiaoMiao2

**Comment by Navid200** [comment, marked as answer]
@psonnera I have seen this before.  But, I don't remember if it is an open issue or an artifact.

**Reply by psonnera** [reply]
This is something I really don't understand. In the case above it's also a sensor related error.
